docs/guide/deployer.md

```diff
@@ -42,3 +45,29 @@ Smart contracts must be stored in `assets` folder.
 The main difference between deploying an ASA and SSC is that ASA takes `asset-name` and `ASADeploymentFlags` as input and SSC takes `smart-contract-names` and `SSCDeploymentFlags` as input.
 
 You can learn more about the flags from [Deployer API](https://scale-it.github.io/algo-builder/);
+
+#### Stateless Smart Contracts
+
+- *Contract Mode:*
+
+  Contract accounts act in a similar fashion to escrow accounts, where when the smart contract is compiled it produces an Algorand address. This address can accept Algos or Algorand ASAs with standard transactions, where the contract’s address is the receiver of the transaction.
+
+   Check our [examples/htlc-pyteal-ts](../examples/htlc-pyteal-ts) project to explore how to deploy Stateless Smart Contracts(lsig). In the file `scripts/deploy.ts`, you will find:
+
+  ```
+  await deployer.fundLsig('htlc.py',
+    { funder: bob, fundingMicroAlgo: 2e6 }, {}, [], scTmplParams);
+  ```
+  `fundLsig` funds the contract account (compiled hash of the smart contract). The function `fundLsig` accepts `pyteal` code too, which provides the functionality of dynamically providing the params before compiling into TEAL code.
```

Comment:
is compiled hash same as contract address ?

Reply:
Yes